runtime/onert/core/include/ir/OpSequence.h

```diff
@@ -66,16 +66,15 @@ class OpSequence
 public:
   void remove(const OperationIndex &index);
 
+  bool exist(const OperationIndex &index) const;
+
```

Comment:
@periannath It is not necessary to move `exist` to `public`.

Reply:
@glistening Thanks! I'll update it.

Reply:
@periannath It's my mistake. It is `OpSequence`, not `OpSequences`.